src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleCss = {};
+  const changeCss = sourceString.split(';');
```

Comment:
Consider trimming the `sourceString` before splitting it to handle any leading or trailing semicolons that may result in empty strings in the `changeCss` array.